### ISSUE TITLE
Dialling back location info?

### ISSUE THREAD

**Issue by adae22** [ParadiseAigo/Tekken-7-Player-Displayer#26]
Hey there,

Thanks for bringing the Player Displayer back up to date so quickly after 5.0 released!

Now, I was wondering if it would be possible to tune the location info down a notch?

I don't know how precise the info is for others or how it affects them, and I understand that the impact and importance of having this feature varies based on where you are (small countries vs big countries).
But as it stands, Player Displayer practically doxes me. 
The fact that I live in a rather small place obviously adds to the problem and makes my case rather specific I suppose, but as of right now streaming my games or partaking in sessions I organize from time to time makes me feel a little uncomfortable to say the least, knowing that someone who can't handle their salt could end up knowing what city I'm in.

I realize that there's other ways of obtaining exactly the same info, but I can't help but feel like it's being offered on a silver platter right now.

So, long story short, perhaps we could bring it down to Country+Region/State?

As always, thanks for the work you're putting into all this and for making it available to all of us, I really appreciate it!

**Comment by ParadiseAigo** [comment]
Okay. If it's making you feel unsafe, I am not going to say no. I still think it's harmless though, all things considered. But as a fan of anonymity and being safe (lol) I can't refuse.

I already updated it.

By the way, if you want to know what other people would have seen if they got matched with you try this link (it's what Player-Displayer uses right now) (6th thing is your city):
ip-api.com/json
or try this one if the above is not readable:
ip-api.com/line

**Comment by adae22** [comment]
Much appreciated!

I think we're on the same page, most of the time it's probably not worth the worry, but I wouldn't want to gamble on it on the off chance that some goofball does come up with some dumb idea.

> By the way, if you want to know what other people would have seen if they got matched with you try this link (it's what Player-Displayer uses right now) (6th thing is your city): ip-api.com/json or try this one if the above is not readable: ip-api.com/line

Yeah, I did something to that effect, which prompted my post.
Like I said, my case is probably a bit of an outlier - if you're living in some major city there's not a whole lot you can do with this info, but if you're in a place of a few thousand it becomes a bit more dicey, haha.

Again, thanks a bunch for your help and work!
